packages/airnode-admin/package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "@api3/airnode-abi": "^0.4.0",
     "@api3/airnode-protocol": "^0.4.0",
+    "@api3/airnode-utilities": "^0.5.0",
```

Comment:
Why is this 0.5?

Reply:
It can be 0.4.0, I just figured that's the version we're going to next.